src/main/java/seedu/medibook/logic/commands/ClearCommand.java

```diff
@@ -18,6 +18,7 @@
     public CommandResult execute(Model model) {
         requireNonNull(model);
         model.setMediBook(new MediBook());
+        model.getContext().setShouldDeleteAllMedicalNotes(true);
         return new CommandResult(MESSAGE_SUCCESS);
     }
 }
```

Comment:
Would it be better if model had a set...() method that called context.set...() instead? It works either way though :D